diff --git a/src/functions.php b/src/functions.php
index 2aab877..214aad6 100644
--- a/src/functions.php
+++ b/src/functions.php
@@ -297,7 +297,7 @@ function _checkTypehint(callable $callback, \Throwable $reason): bool
             break;
         case $type instanceof \ReflectionIntersectionType:
             $isTypeUnion = false;
-        case $type instanceof \ReflectionUnionType;
+        case $type instanceof \ReflectionUnionType:
             $types = $type->getTypes();
             break;
         default:
